packages/components/organisms/f-header/src/components/Logo.vue

```diff
@@ -11,9 +11,11 @@
         }`'>
         <component
             :is="iconComponent"
-            :class="['c-logo-img',
-                     iconClassName,
-                     logoColourModifier]"
+            :class="[
+                $style['c-logo-img'],
+                iconClassName,
+                { [$style['c-logo-img--alt']]: isAltLogo }
```

Comment:
Not sure what the answer is to this one, but do the logo states work ok when JS fails to load?

Just as although we don't aim to support no-JS, but the site should still look styled ok, so would be interested to know if the logo appears ok in that situation?